readthedocs_build/config/config.py

```diff
@@ -14,7 +14,7 @@
     'load', 'BuildConfig', 'ConfigError', 'InvalidConfig', 'ProjectConfig')
 
 
-CONFIG_FILENAMES = ('readthedocs.yml', '.readthedocs.yml')
+CONFIG_FILENAME_REGEX = r'\.?readthedocs.ya?ml'
```

Comment:
I wonder if isn't better to test this regex (because right now we are hardcoding a regex in the tests)

Reply:
Sure, can do that too, is a very easy test, I'm just wondering what will I actually be testing? I can see 4 permutations that the regex need to test, but should I test that it doesn't match something else as well?

Reply:
I think testing matching is enough